lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -13,14 +13,14 @@ byteorder = "1.4.3"
 env_logger = "0.9.0"
 iso7816 = "0.1.0"
 iso7816-tlv = "0.4.2"
-libc = "0.2.121"
-log = "0.4.16"
+libc = "0.2.126"
+log = "0.4.17"
 num-derive = "0.3.3"
-num-traits = "0.2.14"
+num-traits = "0.2.15"
 rand = { version = "0.8.5", features = ["getrandom"] }
 rand_chacha = "0.3.1"
 rsa = "0.6.1"
-rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "6075679e7c9bd8e3c2136a87f097d05c7db3235f" }
+rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "82bcc2d971a336f1550968c9aa617adb03deb2f1" }
```

Comment:
Should we update this to 888b15477d9aaceb83b2afcc73de040a07781d28 while we're here?